app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,45 @@
+
+<div class="col-lg-12" id="signup">
+<h1>Sign up</h1>
+<br>
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="mb-3">
+   
+  <%= f.text_field :name, autofocus: true, class:"form-control",  placeholder:"Enter Your Name" %>
```

Comment:
I have to say that you did a great job on the front end, everything looks good and clean, kudos on that 👍🏻